src/trix/config/text_attributes.coffee

```diff
@@ -3,8 +3,10 @@ Trix.config.textAttributes =
     tagName: "strong"
     inheritable: true
     parser: (element) ->
+      return false if element.tagName in Trix.getBlockTagNames() or element.tagName is "BR"
```

Comment:
In https://github.com/basecamp/trix/pull/263#discussion_r72674150, I meant that you should be able to remove this whole line. The fix is in master so you'll need to merge it in to this branch.